.vscodeignore

```diff
@@ -1,8 +1,18 @@
-.vscode/**
-.vscode-test/**
+# Dir to exclude for packaging
+.github
+.vscode
+.vscode-test
+infra
+src
```

Comment:
`src` folder is not needed?? @.@

Reply:
I found that `src` had some `json` files in `snippet` dir.
I will not add `src` for now. 
Thanks for good catch.